source/reference/template-apps.txt

```diff
@@ -134,52 +134,52 @@ Template Apps Available
 
    * - Manage Database Views
      - ``triggers``
-     - Event-driven :ref:`Database Trigger <database-trigger>` template to update a view in a separate collection.
+     - Event-driven :ref:`Database Trigger <database-trigger>` template to update a view in a separate collection. Does *not* include a front-end app.
 
    * - Flutter + Realm Flutter SDK
      - ``flutter.todo.flex``
-     - Cross-platform application for Android, iOS, Windows and MacOS built with the :ref:`Realm Flutter SDK <flutter-intro>`.
+     - Cross-platform to-do list app for Android, iOS, Windows, MacOS, and Linux written in Dart using the :ref:`Realm Flutter SDK <flutter-intro>`.
```

Comment:
can you add line abt how flitter using sync as well here? 

i think it's not currently included b/c the flutter template app was local only for a short period of time.

Reply:
updated